src/main/java/ClientGet.java

```diff
@@ -0,0 +1,63 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClientGet {
+
+    public static void main(String[] args) throws IOException {
+
+
+        Currencies currenciesTable[] = Currencies.values();
+        List<String> currenciesExchangeRateStringList = new ArrayList<>();
+        List<CurrenciesExchangeRate> currenciesExchangeRateList = new ArrayList<>();
+
+
+        for (int j = 0; j < currenciesTable.length; j++) {
```

Comment:
Use java for each loop instead of normal for - it will be more readable.

Reply:
According to above comment -> you can create something like this:
`for(Currencies first: Currencies.values()) {
    for(Currencies second: Currencies.values()) {
      while(...) {
            ....
            CurrenciesExchangeRate cer = new CurrenciesExchangeRate(first, second, calculatedRate);
            ...
      }
`

Reply:
Bad formatting :( but copy to intellij to see it in more clear way.